### DISCUSSION TITLE
Ganache deployed on kubernetes

### DISCUSSION THREAD

**Discussion by TejPandey** [ConsenSys-archive/ganache#4524] [Q&A]
Hi All,
I am facing issue when I am trying to deploy the ganache-cli on kubernetes. When I am deploying with persistent volume with dererministic=true, I can see in logs that it generates 10 accounts with balances. But balances are coming to metamask when imported the account. Although deployment without persistent volume is working fine and balances are coming in metamask.
Below is the pod describe result.
<img width="1125" alt="image" src="https://github.com/trufflesuite/ganache/assets/20352666/44e46607-6ef3-4a28-9864-72a07ec508b4">

**Comment by davidmurdoch** [comment]
Try using the ganache package/image, not ganache-cli.
